### DISCUSSION TITLE
I can't download old version for Windows XP

### DISCUSSION THREAD

**Discussion by InfoLibre** [sumatrapdfreader/sumatrapdf#3505] [Q&A]
I'd like to download this version: https://www.sumatrapdfreader.org/dl/rel/3.1.2/SumatraPDF-3.1.2-install.exe but I've got this Cloudflare error:

![Capture d’écran du 2023-05-21 08-51-37](https://github.com/sumatrapdfreader/sumatrapdf/assets/3955499/8db716d3-e784-4bbb-a180-2714ebfab9b6)

Cloudflare asks to create a public bucket if I click on the link on this error page: https://developers.cloudflare.com/r2/buckets/public-buckets/

PS: could someone add real links on https://www.sumatrapdfreader.org/download-prev, not just text?

**Comment by GitHubRulesOK** [comment]
all hyperlinks to download are text just those texts are not as leachable 
and just like you robots will not be able to download because something has gone on walkabout/holiday/sleep 

I don't know why only the 3.1.2 and earlier are non functional but 3.2 are available.

If you look at Issues you will see its been a problem for a while and suggested alternative for 3.1.2 is here https://github.com/sumatrapdfreader/sumatrapdf/issues/3474

**Comment by InfoLibre** [comment]
I don't want to download outside the official website. Nobody could solve this problem and add normal links?

**Comment by InfoLibre** [comment]
I found it on https://web.archive.org/web/20160816164120/http://www.sumatrapdfreader.org/download-free-pdf-viewer-fr.html

**Reply by GitHubRulesOK** [reply]
that's not much different to the suggested links anyway for comparison those listed untested copies in French page are
64 bit
https://kjkpub.s3.amazonaws.com/sumatrapdf/rel/SumatraPDF-3.1.2-64-install.exe
https://kjkpub.s3.amazonaws.com/sumatrapdf/rel/SumatraPDF-3.1.2-64.zip
32 bit
https://kjkpub.s3.amazonaws.com/sumatrapdf/rel/SumatraPDF-3.1.2-install.exe
https://kjkpub.s3.amazonaws.com/sumatrapdf/rel/SumatraPDF-3.1.2.zip

**Comment by kjk** [comment]
Sorry it took so long. Should be fixed now.
